src/lib/validation.ts

```diff
@@ -55,32 +57,48 @@ export function checkString(s: unknown): string | void {
 	return s;
 }
 
+/**
+ * checkBooleanish
+ *
+ * Given an unknown value, determine if it can be evaluated to truthy/falsey.
+ *
+ */
+export function checkBooleanish(v: unknown): boolean {
+	return checkTruthy(v) || checkFalsey(v);
+}
+
 /**
  * checkTruthy
  *
- * Given a value which can be a string, boolean or number, return a boolean
- * which represents if the input was truthy
+ * Given an unknown value, determine if it evaluates to true.
+ *
  */
-export function checkTruthy(v: unknown): boolean | undefined {
+export function checkTruthy(v: unknown): boolean {
 	if (_.isString(v)) {
 		v = v.toLowerCase();
 	}
-	switch (v) {
-		case '1':
-		case 'true':
-		case true:
-		case 'on':
-		case 1:
-			return true;
-		case '0':
-		case 'false':
-		case false:
-		case 'off':
-		case 0:
-			return false;
-		default:
-			return;
+	return (
+		TRUTHY.filter((t) => {
+			return t === v;
+		}).length > 0
+	);
+}
+
+/**
+ * checkFalsey
+ *
+ * Given an unknown value, determine if it evaluates to false.
+ *
+ */
+export function checkFalsey(v: unknown): boolean {
+	if (_.isString(v)) {
```

Comment:
I generally prefer native versions to lodash when they're equivalent length/functionality, ie
```suggestion
	if (typeof v === 'string') {
```
here, but eg still things like `_.sum` that do things with no native equivalent. The reason being roughly that the native versions have the ability to be optimized better, are more standard nowadays, and are perfectly safe when you're not dealing with unknown types - lodash helps by making sure things like eg `.map` are always available (it's from the days when you'd often support browsers that didn't have such things natively available), and making sure it works consistently across null/arrays/objects (but with typescript we should be able to rely on having the right data types, and if not it's a bug we should fix rather than hide)

Reply:
I favor the std lib as well. Thanks for pointing this instance out.